### ISSUE TITLE
flake8 `E305` error in `setup.py:42:1` during build

### ISSUE THREAD

**Issue by gfrogat** [phovea/phovea_clue#131]
* Release number or git hash: 921409de073f4329c09a4602622e87d816de266f
* Web browser version and OS: macOS 10.13.1
* Environment (local or deployed): local

### Steps to reproduce

1. clone project master branch and install dependencies
`git clone https://github.com/phovea/phovea_clue.git`
2. run tests
`npm test`

### Observed behavior
* flake8 error due to missing blank line in line 42 of file `setup.py`
```
./setup.py:42:1: E305 expected 2 blank lines after class or function definition, found 1
```

### Expected behavior
* tests run without any errors

**Comment by thinkh** [comment]
@gfrogat you are working with the master branch. We usually work with the develop branch.

@sgratzl Can you/we release the deploy branch as new stable version to avoid confusions?

**Comment by thinkh** [comment]
Should be fixed with the most recent version v2.3.0.
